<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWifi</class>
 <widget class="QWidget" name="TestWifi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="TbConnect">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>101</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>链接网络</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>40</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索到的网络：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>741</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TbScanning">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索网络</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TbOpen">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wifi-ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbPasswd">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LePasswd">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TbBack">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>28</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ping -c 4 www.google.com.cn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Send">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Clear">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>410</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
